jekyll/_cci2/executor-intro.adoc

```diff
@@ -0,0 +1,346 @@
+  build: # name of your job
+    resource_class: 'windows.medium'
+    machine:
+      image: 'windows-server-2022-gui:current'
+      shell: 'powershell.exe -ExecutionPolicy Bypass'
+    steps:
+      # Commands are run in a Windows virtual machine environment
+        - checkout
+        - run: Write-Host 'Hello, Windows'
+----
+--
+
+[.tab.windowsblock.Server]
+--
+[,yml]
+----
+version: 2.1
+
+jobs:
+  build: # name of your job
+    machine:
+      image: windows-default
+    steps:
+      # Commands are run in a Windows virtual machine environment
+        - checkout
+        - run: Write-Host 'Hello, Windows'
+----
+--
+
+Find out more about the Windows execution environment in the xref:using-windows#[Using the Windows Execution Environment] page. See link:https://circleci.com/developer/orbs/orb/circleci/windows[the Windows orb page in the developer hub] for the list of options available in the Windows orb.
+
+[#gpu]
+== GPU
+
+To access the GPU execution environment, use one of the following:
+
+* The Windows orb and specify the GPU-enabled executor.
+* The `machine` executor and specify a Linux or Windows GPU-enabled image.
+
+For a full list of `machine` images, see the link:https://circleci.com/developer/images?imageType=machine[CircleCI Developer Hub].
+
+NOTE: It is not possible to run a GPU build on CircleCI server.
+
+[.tab.gpublock.Linux]
+--
+[,yml]
+----
+version: 2.1
+
+jobs:
+  build:
+    machine:
+      image: linux-cuda-12:default
+    resource_class: gpu.nvidia.medium
+    steps:
+      - run: nvidia-smi
+----
+--
+
+[.tab.gpublock.Windows_without_orb]
+--
+[,yml]
+----
+version: 2.1
+
+jobs:
+  build:
+    machine:
+      image: windows-server-2019-cuda
+    resource_class: gpu.nvidia.small
+    steps:
+      - run: nvidia-smi
+----
+--
+
+[.tab.gpublock.Windows_with_orb]
+--
+[,yml]
+----
+version: 2.1
+
+orbs:
+  win: circleci/windows@4.1.1
+
+jobs:
+  build:
+    executor: win/server-2019-cuda
+    steps:
+      - run: 'Write-Host ''Hello, Windows'''
+----
+--
+
+Find out more about the GPU execution environment on the xref:using-gpu#[Using the GPU Execution Environment] page.
+
+[#arm]
+== Arm VM
+
+To access the Arm VM execution environment, use the `machine` executor as detailed below, and specify either the `arm.medium` or the `arm.large` resource class. For a full list of `machine` images, see the link:https://circleci.com/developer/images?imageType=machine[CircleCI Developer Hub].
+
+[.tab.armblock.Cloud]
+--
+[,yml]
+----
+# .circleci/config.yml
+version: 2.1
+
+jobs:
+  build-medium:
+    machine:
+      image: ubuntu-2004:202101-01
+    resource_class: arm.medium
+    steps:
+      - run: uname -a
+      - run: echo "Hello, Arm!"
+
+  build-large:
+    machine:
+      image: ubuntu-2004:202101-01
+    resource_class: arm.large
+    steps:
+      - run: uname -a
+      - run: echo "Hello, Arm!"
+
+workflows:
+  build:
+    jobs:
+      - build-medium
+      - build-large
+----
+--
+
+[.tab.armblock.Server_3]
```

Comment:
Is there a way to specify that, by default, this is AWS only? Our server customers in GCP can't use ARM images.